cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -483,7 +483,7 @@ object SnappyContextPerConnection {
     val context = connectionIdMap.get(connectionID)
     if (context != null) context
     else {
-      val snc = SnappyContext.getOrCreate(null: SparkContext).newSession()
+      val snc = SnappyContext(null: SparkContext).newSession()
```

Comment:
That's correct Sumedh, as I have given an example, two different SqlContext with different conf will create confusion even now with Spark. I think their UI is not suited for multiple live sessions/connection etc.. Probably we need to work for an extension to Spark UI.